packages/epo-react-lib/src/form/HorizontalSlider/HorizontalSlider.tsx

```diff
@@ -0,0 +1,98 @@
+import { useState, FunctionComponent } from "react";
+import * as Styled from "./styles";
+
+type SliderValue = number | readonly number[];
+
+interface HorizontalSliderProps {
+  min?: number;
+  max?: number;
+  step?: number;
+  value: number | number[];
+  label: string;
+  minLabel?: string;
+  maxLabel?: string;
+  labelledbyId?: string;
+  onChangeCallback: (value: number | number[], label: string) => void;
+  color?: string;
+  darkMode?: boolean;
+}
+
+const HorizontalSlider: FunctionComponent<HorizontalSliderProps> = ({
+  min = 0,
+  max = 100,
+  step = 1,
+  value,
+  onChangeCallback,
+  label,
+  minLabel,
+  maxLabel,
+  labelledbyId,
+  color = "transparent",
+  darkMode = false,
+}) => {
+  const [showThumbLabels, setShowThumbLabels] = useState(false);
+  const hasDoubleHandles = Array.isArray(value) && value.length > 1;
```

Comment:
double handles are part of `react-slider` natively, and can be determined in the component by checking if the value provided is an array longer than 1 item.